.github/workflows/publish-to-pypi.yml

```
@@ -0,0 +1,61 @@
+# Workflow for publishing archives to PyPI
+name: Publish to PyPI
+
+on:
+  # Runs for pull requests for testing purposes
+  pull_request:
+    branches:
+      - master
```

We don't want to trigger the workflow for pushes and pull requests.
```suggestion
  # Runs for pull requests for testing purposes
  #pull_request:
  #  branches:
  #    - master
```